cog.c

```diff
@@ -345,7 +358,12 @@ on_create_view (CogShell *shell, void *user_data G_GNUC_UNUSED)
     // must have succeeded in providing a WebKitWebViewBackend* instance.
     if (!view_backend)
         g_error ("Could not instantiate any WPE backend.");
-#endif
+
+    struct wpe_view_backend* wpe_backend =
+        webkit_web_view_backend_get_wpe_backend (view_backend);
+    wpe_view_backend_dispatch_set_device_scale_factor (wpe_backend,
```

Comment:
This should be guarded inside a #if HAVE_DEVICE_SCALING block, no?

Reply:
Good catch, thanks! I'll add the guards.